docs/tinlake/contracts/contracts.md

```diff
@@ -135,8 +135,3 @@ The `Assessor` contract is responsible for tracking a few different metrics used
 
 #### Tranche Token Price: `calcTokenPrice(address tranche)`
 The token price can be calculated in a number of ways by the assessor. For example it can take the outstanding debt from all borrowers and discount it by a factor (as implemented in the default). It could also rely on an external price oracle to provide a market price of the tokens.
-
-#### Interest
```

Comment:
We should address this in the assessor. Can you create a github issue for this so we don't forget?